pkg/rbd/controllerserver.go

```diff
@@ -87,10 +52,17 @@ func (cs *ControllerServer) validateVolumeReq(req *csi.CreateVolumeRequest) erro
 	if req.VolumeCapabilities == nil {
 		return status.Error(codes.InvalidArgument, "Volume Capabilities cannot be empty")
 	}
+	options := req.GetParameters()
+	if value, ok := options["clusterID"]; !ok || len(value) == 0 {
+		return status.Error(codes.InvalidArgument, "Missing or empty cluster ID to provision volume from")
```

Comment:
s/Missing/missing

Reply:
I thought we are discussing this [here](https://github.com/ceph/ceph-csi/pull/312#discussion_r282535932), IOW, do you want me to change the strings that I added in this patch to capitalized first letter OR are we dealing with this in a separate patch as the code base needs a revision on the same? Please let me know so that I can handle it appropriately.